### ISSUE TITLE
New version doesn't allow wrapping of text in source control commit message

### ISSUE THREAD

**Issue by LoyalPotato** [microsoft/vscode#142440]
Issue Type: <b>Bug</b>

All I think is needed is to just have version 1.64.0. I tried it on 1.63.x and it still works

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details><summary>Extensions (35)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.2.2
project-manager|ale|12.4.0
gitflow-actions-sidebar|ard|0.5.2
quitcontrol-vscode|art|4.0.0
npm-intellisense|chr|1.4.0
vscode-markdownlint|Dav|0.46.0
vscode-eslint|dba|2.2.2
githistory|don|0.6.19
xml|Dot|2.5.1
gitlens|eam|11.7.0
vscode-html-css|ecm|1.10.2
vsc-material-theme|Equ|33.2.2
prettier-vscode|esb|9.2.0
auto-close-tag|for|0.5.13
auto-rename-tag|for|0.1.9
vscode-pull-request-github|Git|0.34.3
todo-tree|Gru|0.0.215
output-colorizer|IBM|0.1.2
axios-snippets|loy|1.0.2
remote-wsl|ms-|0.63.13
docthis|oou|0.8.2
advanced-new-file|pat|1.2.2
material-icon-theme|PKi|4.12.1
vscode-css-peek|pra|4.2.0
vscode-yaml|red|1.4.0
LiveServer|rit|5.7.4
code-settings-sync|Sha|3.4.3
vscode-taskexplorer|spm|2.9.1
autoimport|ste|1.5.4
errorlens|use|3.4.1
vscode-todo-highlight|way|1.0.5
JavaScriptSnippets|xab|1.8.0
vscode-surround|yat|1.2.2
markdown-all-in-one|yzh|3.4.0
html-css-class-completion|Zig|1.20.0

(1 theme extensions excluded)

</details>

**Comment by gjsjohnmurray** [comment]
Doesn't reproduce for me:

![junk](https://user-images.githubusercontent.com/6726799/152878194-21ba2ba6-293d-4713-a2bd-aa8bade55f1e.gif)

Version: 1.64.0 (user setup)
Commit: 5554b12acf27056905806867f251c859323ff7e9
Date: 2022-02-03T04:22:20.678Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000

Maybe test with all of your extensions disabled.

**Comment by LoyalPotato** [comment]
I'm sorry, it was happening in my macbook, I just submitted through the windows version. Here are the informations on vscode about in that system:

Version: 1.64.0
Commit: 5554b12acf27056905806867f251c859323ff7e9
Date: 2022-02-03T04:20:17.224Z (4 days ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.2.0

But I've since restarted vscode on it, and it doesn't seem to have that problem anymore. Do I close the issue?
